util/string_view.inl

```diff
@@ -141,6 +141,15 @@ inline std::string_view slice_suffix(std::string_view view, std::size_t size)
   return view;
 }
 
+inline bool starts_with(std::string_view view, std::string_view prefix)
+{
+  if (view.length() < prefix.length()) {
+    return false;
+  }
```

Comment:
Seems `substr` already handles this case gracefully:

```
Returns a view of the substring [pos, pos + rcount), where rcount is the smaller of count and size() - pos.
```
from https://en.cppreference.com/w/cpp/string/basic_string_view/substr

So consider replacing this check with a "// note: view.substr() already handles the case where view.length() > prefix.length()". or something similar (this should eliminate the run-time cost of the redundant check)

Reply:
done